api/set_integer_now_func.md

```diff
@@ -11,40 +11,54 @@ api:
 
 # set_integer_now_func()
 
-This function is only relevant for hypertables with integer (as opposed to
-TIMESTAMP/TIMESTAMPTZ/DATE) time values. For such hypertables, it sets a
-function that returns the `now()` value (current time) in the units of the time
-column. This is necessary for running some policies on integer-based tables.
-In particular, many policies only apply to chunks of a certain age and a
-function that returns the current time is necessary to determine the age of a
-chunk.
+Override the [`now()`](https://www.postgresql.org/docs/16/functions-datetime.html) date/time function used to
+set the current time in the integer `time` column in a hypertable. Many policies only apply to 
+[chunks][hash-partitions] of a certain age. `integer_now_func` determines the age of each chunk.
```

Comment:
Not sure why you call this "hash partitioning" since it is linking to general partitioning functions. It's only the link name, but so it is not visible to users, but it is a little confusing when you you want to edit the documentation.

Reply:
I copied and pasted the link from another page :-). 


Updated to match the link text.